CHANGELOG.md

```diff
@@ -51,7 +51,7 @@
 - PR #68 Use YYMMDD tag in nightly build of cuspatial python
 - PR #97 Drop `cython` from run requirements
 - PR #82 Added update-version.sh
-- PR #86 Add Shapefile reader for polygons
```

Comment:
I think a weird merge happened. Or looks like you switched the target branch. You might want to merge branch-0.12 into your branch or maybe just rebase.